### PR TITLE
Generalize airtable

### LINKED ISSUE #566: generalize Airtable -> Redux code
Make an array somewhere (`utils` maybe?) of the tables we want to pull out of Airtable. Change our code so that all those tables are downloaded and put into redux. This will make it MUCH easier to add or rename future tables.

### PR DESCRIPTION
**Pull request by sastels**
Resolves #566 
Make it easier to load a new table from AirTable (or rename a current one).

### CODE REVIEW

File: store.js [hunk 1]
```diff
@@ -55,18 +54,15 @@ export const reducer = (state = initialState, action) => {
...
-        needs: action.data.needs || state.needs,
-        examples: action.data.examples || state.examples,
         favouriteBenefits:
-          action.data.favouriteBenefits || state.favouriteBenefits,
-        translations: action.data.translations || state.translations,
-        areaOffices: action.data.areaOffices || state.areaOffices
+          action.data.favouriteBenefits || state.favouriteBenefits
+      };
+      airtableConstants.tableNames.forEach(tableName => {
+        newState[tableName] = action.data[tableName] || state[tableName];
```
Comment (maxneuvians) [inline store.js:64]: 🎉

File: utils/airtable_es2015.js [hunk 2]
```diff
@@ -33,13 +34,19 @@ var fetchTableFromAirtable = async function fetchTableFromAirtable(table) {
...
+    dataStore[tableName] = await fetchTableFromAirtable(tableName);
+  });
+
+  // dataStore.benefits = await fetchTableFromAirtable("benefits");
+  // dataStore.eligibilityPaths = await fetchTableFromAirtable("eligibilityPaths");
+  // dataStore.needs = await fetchTableFromAirtable("needs");
+  // dataStore.examples = await fetchTableFromAirtable("examples");
+  // dataStore.translations = await fetchTableFromAirtable("translations");
+  // dataStore.areaOffices = await fetchTableFromAirtable("areaOffices");
+  // dataStore.timestamp = await Date.now();
```
Comment (maxneuvians) [inline utils/airtable_es2015.js]: Can this line still go in? We need it for a Log in `server.js`

**Review by maxneuvians — APPROVED** [review]
LGTM, just one small request re: date logging

File: utils/airtable_es2015.js (hunk 2 above)
Comment (sastels) [reply utils/airtable_es2015.js]: whoops, to many comments. 👍

**Pull request merged**